openfasoc/common/drc-lvs-check/magic_commands.tcl

```diff
@@ -8,6 +8,13 @@
 #   tech load $::env(TECH)
 # }
 
+# gds flatglob commands removes the drc errors caused by hierarchy that magic can't process.
+# gds faltglob *capacitor_test_nf* is specific to ldo-gen
+gds flatglob *$$*
```

Comment:
@alibillalhammoud same as in @lulu9312 's PR. What are these errors?

Reply:
@lulu9312 was working on DRC errors and we have been syncing our changes with each other. These are same commits/changes from Pranav's PR. Pranav can follow up on this question.